js/hoverzoom.js

```diff
@@ -340,8 +363,11 @@ var hoverZoom = {
                 //hz.hzImg.css('visibility', 'visible');
 
                 // image natural dimensions
-                imgDetails.naturalWidth = imgFullSize.width() * zoomFactor;
-                imgDetails.naturalHeight = imgFullSize.height() * zoomFactor;
+                //imgDetails.naturalWidth = imgFullSize.width() * options.zoomFactor;
+                //imgDetails.naturalHeight = imgFullSize.height() * options.zoomFactor;
+                imgDetails.naturalWidth = (imgFullSize[0].naturalWidth ? imgFullSize[0].naturalWidth : imgFullSize.width()) * options.zoomFactor;
+                imgDetails.naturalHeight = (imgFullSize[0].naturalHeight ? imgFullSize[0].naturalHeight : imgFullSize.height()) * options.zoomFactor;
```

Comment:
The only part I'm not sure about is here. I think you gotta use zoomFactor and not options.zoomFactor.
I explicitly take a snapshot of options.zoomFactor at the the top of loadFullSizeImage and then later modify it inside documentOnMouseWheel. This will break mousewheel zooming of the locked image I believe